code/modules/goonchat/browserOutput.dm

```diff
@@ -115,12 +115,15 @@ GLOBAL_DATUM_INIT(iconCache, /savefile, new("tmp/iconCache.sav")) //Cache of ico
 		data = json_encode(data)
 	C << output("[data]", "[window]:ehjaxCallback")
 
-/datum/chatOutput/proc/sendMusic(music, pitch)
+/datum/chatOutput/proc/sendMusic(music, list/extra_data)
 	if(!findtext(music, GLOB.is_http_protocol))
 		return
 	var/list/music_data = list("adminMusic" = url_encode(url_encode(music)))
-	if(pitch)
-		music_data["musicRate"] = pitch
+
+	if(extra_data && extra_data.len)
```

Comment:
```suggestion
	if(extra_data?.len)
```

Reply:
@ninjanomnom This always works? Like it checks if it is not null and then checks the "subvalue"?

Reply:
it's equivalent

Reply:
That's cool! I'll implement that right away